plugins/inputs/sqlserver/README.md

```diff
@@ -118,6 +118,14 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
 
 [CONFIGURATION.md]: ../../../docs/CONFIGURATION.md#plugins
 
+## Secret-store support
+
+This plugin supports secrets from secret-stores for the `servers` option.
+See the [secret-store documentation][SECRETSTORE] for more details on how
+to use them.
+
```

Comment:
I missed this PR, but what about adding an example to some of the more complex ones?
```suggestion
This plugin supports secrets from secret-stores for the `servers` option.
 See the [secret-store documentation][SECRETSTORE] for more details on how
 to use them.

Example: `Server=192.168.1.10;Port=1433;User Id=@{my_secret_store:sqlserver_user};Password=@{my_secret_store:sqlserver_pass};app name=telegraf;log=1;`
```

Reply:
Feel free to put up a PR!